default.go

```diff
@@ -109,7 +113,20 @@ func IsHostname(str string) bool {
 			valid = false
```

Comment:
Please consider removing valid variable and simply do this

```suggestion
			return false
```

So you can remove the if

https://github.com/go-openapi/strfmt/pull/150#discussion_r1996304984

Reply:
yes. Much better. That in there and I omitted this small refactoring. Thanks